pyaedt/modeler/actors.py

```diff
@@ -140,33 +139,31 @@ def insert(self, app, motion=True):
 
 
 class Bird(Actor, object):
-    """Provides an instance of an actor.
+    """Provides an instance of a bird.
 
-    This class is derived from :class:`MultiPartComponent`.
+    This class is derived from :class:`pyaedt.modeler.multiparts.MultiPartComponent`.
 
     .. note::
-       Motion is always forward in the X-direction.
+       Motion is always forward in the X-axis direction.
 
     Parameters
     ----------
-    bird_folder : str, required
+    bird_folder: str
```

Comment:
```suggestion
    bird_folder : str
```

Reply:
@PipKat Alex's PR should have fixed these issues on the main branch, but it did not fix the ones introduced in your branch.